components/status/HeatMapColorBar.vue

```diff
@@ -0,0 +1,212 @@
+<template>
+  <div>
+    <div id="parent-container">
+      <div
+        id="startup-heatmap"
+        class="div__heatmap-gradient-holder"
+        :style="value"
+      ></div>
+    </div>
+    <!-- prettier-ignore -->
+    <span class="span__heatmap-scale-higher-value">{{ upper_range }} {{ units }}</span>
+    <!-- prettier-ignore -->
+    <span class="span__heatmap-scale-lower-value" :style="top_shift">{{ lower_range }} {{ units }}</span
+    >
+  </div>
+</template>
+<script>
+/**
```

Comment:
it looks like you meant to put a docstring in these comments (specifically regarding the explanation of how the two separate props for the gradient are handled)---is that missing?

Reply:
Done